api/src/controllers/team.js

```diff
@@ -220,19 +223,16 @@ async function deployToGHPages(req, res) {
     teamPublications: publications,
   };
 
-  try {
-    const schollyResponse = await axios({
-      url: schollyHost + '/deploy/' + teamId,
-      method: 'post',
-      data: body,
-      headers: {
-        'Content-Type': 'application/json',
-      },
-    });
-    return res.status(200).json(schollyResponse.data);
-  } catch (err) {
-    console.log(err);
-  }
+  await axios({
+    url: schollyHost + '/deploy/' + teamId,
+    method: 'post',
+    data: body,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
```

Comment:
This may not be as readable as a simple `axios.post`. The json content type will be automatically applied if the body is a JS object: https://masteringjs.io/tutorials/axios/post-json